packages/xrpl/test/integration/requests/bookOffers.test.ts

```diff
@@ -35,6 +35,7 @@ describe('book_offers', function () {
       const response = await testContext.client.request(bookOffer)
 
       const expectedResponse: BookOffersResponse = {
+        api_version: 2,
```

Comment:
can we remove these `api_version: 2` lines in the integration tests? In the future when we upgrade to V3, we'll need to update all these lines again.

I understand that it defaults to V2, if no version is specified

Reply:
It's better to keep this because it will serve as a safeguard/reminder that whenever we change the default `api_version` then we should also update the integ tests.